doc/modules/mod_vcard.md

```diff
@@ -3,47 +3,139 @@ This module provides support for vCards, as specified in [XEP-0054: vcard-temp](
 
 ### Options
 
-* `iqdisc` (default: `one_queue`)
-* `host` (string, default: `"vjud.@HOST@"`): Domain of the vCard User Directory, used for searching.
- `@HOST@` is replaced with the domain(s) supported by the cluster.
-* `search` (boolean, default: `true`): Enables/disables the domain set in the previous option.
- `false` makes searching for users impossible.
-* `backend` (atom, default: `mnesia`): vCard storage backend.
- Valid values are `ldap`, `rdbms`, `riak` and `mnesia`.
- **Warning:** LDAP backend is read-only.
-* `matches` (`infinity` or positive integer, default: 30): Maxmimum search results to be returned to the user.
+#### `modules.mod_vcard.iqdisc.type`
+* **Syntax:** string, one of `"one_queue"`, `"no_queue"`, `"queues"`, `"parallel"`
+* **Default:** `"no_queue"`
+
+Strategy to handle incoming stanzas. For details, please refer to
+[IQ processing policies](../../advanced-configuration/Modules/#iq-processing-policies).
+
+#### `modules.mod_vcard.host`
+* **Syntax:** string
+* **Default:** `"vjud.@HOST@"`
+* **Example:** `host = "vjud.@HOST@"`
+
+Domain of the vCard User Directory, used for searching.
+`@HOST@` is replaced with the domain(s) supported by the cluster.
+
+#### `modules.mod_vcard.search`
+* **Syntax:** boolean
+* **Default:** `true`
+* **Example:** `search = false`
+
+Enables/disables the domain set in the previous option. `false` makes searching for users impossible.
+
+#### `modules.mod_vcard.backend`
+* **Syntax:** string, one of `"ldap"`, `"rdbms"`, `"riak"`, `"mnesia"`
+* **Default:** `"mnesia"`
+* **Example:** `backend = "rdbms"`
+
+vCard storage backend. **Warning:** LDAP backend is read-only.
+
+#### `modules.mod_vcard.matches`
+* **Syntax:** non-negative integer or the string `"infinity"`
+* **Default:** `30`
+* **Example:** `matches = 10`
+
+Maximum search results to be returned to the user.
 
 ##### LDAP-specific options
 
-* `ldap_pool_tag`, `ldap_base`, `ldap_uids`, `ldap_filter`, `ldap_deref`:
-  These options are the same as for the [LDAP authentication module](../authentication-backends/LDAP-authentication-module.md#configuration-options).
 
-* `ldap_vcard_map` (list of `{VCardField, LDAPPattern, [LDAPField]}`, default: see description): Mappings between VCard and LDAP fields. For the default setting, please see `[MongooseIM root]/src/mod_vcard_ldap.erl`, line 74.
+###### `ldap_pool_tag`, `ldap_base`, `ldap_uids`, `ldap_filter`, `ldap_deref`
```

Comment:
Here and in the options below, why not following the convention of prefixing the options with the whole path?

Reply:
Maybe something like:

```
The following options are the same as for the [LDAP authentication module](../authentication-backends/LDAP-authentication-module.md#configuration-options):
- `ldap_pool_tag`,
- `ldap_base`,
- `ldap_uids`,
- `ldap_filter`,
- `ldap_deref`
```

The best solution would be to capture them both with the same handler function and make and LDAP subsection, but rather not in this PR.